### PR TITLE
Feed Example: Convert images to inline SVG

### LINKED ISSUE #2628: Feed Display Example Broken Star Images
There are broken links in the [feed example](https://www.w3.org/WAI/ARIA/apg/patterns/feed/examples/feed/) on the [dedicated display page](https://www.w3.org/WAI/ARIA/apg/patterns/feed/examples/feed-display.html): there should be images representing the number of stars each restaurant has received.

We discussed the issue in the task force and we think the issue is caused by the fact that the links to the images are generated by JavaScript, and we think a good course of action would be to use an inline SVG instead of an external link.

### PR DESCRIPTION
**Pull request by frozenzia**
Fixes #2628
Possibly also w3c/wai-aria-practices#190 ? Was not clear to me the relation between aria-practices and wai-aria-practices.

Once this is OK, need to remember to deal with issue w3c/wai-aria-practices#225. Again, not sure about the relation, sorry if it shouldn't be mentioned here.

Ah, couple of points.
1) as suggested, I used radio-rating.html as "inspiration" for this fix. The CSS for that file included the following bits:
```
.restaurant-rating svg {
  forced-color-adjust: auto;
  touch-action: pan-y;
}

.restaurant-rating svg .star-none {
  stroke-width: 3px;
  stroke: #f8951d;
  fill-opacity: 0;
}
```
..., which I did **not** include / copy for the feed example, as they didn't seem necessary. But supposing that I've misunderstood something, I could be wrong about that.

2) Also just realized I've left .restaurant-star-img selector in the CSS and it's completely unnecessary.

### Preview feed example in compare branch

[Preview of feed example display page](https://deploy-preview-234--aria-practices.netlify.app/aria/apg/patterns/feed/examples/feed-display.html)
### Review checklist

* N/A: [Editorial review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#editorial_review)
* N/A: [ ] [Functional review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#functional_review)
* [x] [Visual design review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#design_review) by reviewer_id
* [x] [Accessibility review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#accessibility_review) by reviewer_id
* [x] [Code review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#code_review) by reviewer_id
 * N/A: [Test review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#test_review) by reviewer_id

___
[WAI Preview Link](https://deploy-preview-234--aria-practices.netlify.app/ARIA/apg) _(Last built on Wed, 28 Jun 2023 07:48:36 GMT)._

### CODE REVIEW

File: content/patterns/feed/examples/js/feedDisplay.js [hunk 1]
```diff
@@ -211,14 +211,31 @@ aria.FeedDisplay.prototype.renderItemData = function (itemData) {
...
       itemData.rating +
-      '.png">' +
+      '">' +
+      '<svg role="none" xmlns="http://www.w3.org/2000/svg" viewBox="38 -5 100 48" width="90" height="25">' +
+      '<defs>' +
+      '<g id="star">' +
+      '<polygon points="2.0,13.4 11.7,20.5 8.0,31.1 17.7,24.8 27.4,31.9 23.7,20.5 33.4,13.4 21.4,13.4 17.7,2.0 14.0,13.4" />' +
+      '</g>' +
+      '</defs>' +
+      '<g aria-label="one star" class="star-1 star-2 star-3 star-4 star-5">' +
```
Comment (mcking65) [inline content/patterns/feed/examples/js/feedDisplay.js:223]: I don't know why, but this isn't working. When I look at the preview with JAWS in Chrome, the ratings graphics are not showing up.

Comment (frozenzia) [reply content/patterns/feed/examples/js/feedDisplay.js:223]: I presume the inline SVG in the radio-rating example *is* working? I'll see what I can figure out.

Comment (frozenzia) [reply content/patterns/feed/examples/js/feedDisplay.js:223]: Ah, I think I'm misunderstanding your comment, but just checked the VoiceOver output from the end-result code and that is also def. NOT working properly.

Comment (frozenzia) [reply content/patterns/feed/examples/js/feedDisplay.js]: Aria-label fixed in latest commit. Not 100% sure if that solves the issue you were having or not.

Comment (mcking65) [reply content/patterns/feed/examples/js/feedDisplay.js]: Yes, you fixed it!!!

In this Tuesday's meeting, we'll work on getting reviewers assigned.

**Comment by jnurthen** [comment]
IMO the SVG should recolour based on high contrast mode. It is not currently doing this.

**Comment by frozenzia** [comment]
@jnurthen, alright, I just tried to mimic the existing images. I'll look into "high contrast mode" and try to come up w/better colors.

**Comment by frozenzia** [comment]
@jnurthen, understand your comments better now that I learned what "high contrast mode" even is. Latest commit *should* fix it if I've understood things right. I don't have a Windows machine to test on, and actually Mac's "Invert colours" does not seem to trigger the media-query (for "forced-colors") I added, but is that okay? It does work as intended if I use Chrome and "Rendering" settings to manually emulate "forced-colors". If I've understood properly, that should then also work for Windows High Contrast Mode.

**Review by a11ydoer — APPROVED** [review]
The code change looks good to me.

**Review by jongund — APPROVED** [review]
High contrast mode on windows works great on the inline SVG images.

**Review by andreancardona — APPROVED** [review]
Code review looks good

**Comment by mcking65** [comment]
Thank you @frozenzia!!

**Pull request merged**